contents/tutorials/holdout-testing.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to do holdout testing
+date: 2023-07-05
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-12.png
+tags: []
+---    
+
+Holdout testing is a type of [A/B testing](/docs/experiments) that keeps a "holdout" group for the long term after the completion of the main part of the experiment. This ensures the experiment doesn’t have negative long term effects or interaction with other experiments. This is particularly important in high-risk products or features like finance or your signup page.
+
+In this tutorial, we show you how to set up a holdout test in PostHog.
+
+## Creating an experiment
+
+To create a holdout test, first, create an experiment. 
+
+1. Go to the [experiments tab](https://app.posthog.com/experiments) in PostHog and click "New experiment." 
+2. Name your experiment, set a key, and add a description. Your name and description should make it clear this is a holdout test for future reference. 
+3. Add another variant named "holdout" to the existing "test" and "control" variants. 
+4. Fill out the rest of your experiment with participants, goals, and acceptable. 
+5. Save it as a draft.
+
+Once saved, go to [your feature flags](https://app.posthog.com/feature_flags) and edit the flag (you can find it with the key you set in your experiment). Here, set the variant rollout percentages for the "test" and "control" to 45% each, and the rollout for "holdout" to 10%. Once done, press save.
+
+![Original split feature flags](../images/tutorials/holdout-testing/split.png)
+
+## Implement the experiment
+
+With the experiment and flag all setup, you can implement it in your app. Add the flag evaluation logic so that only the "test" variant shows the changed component or code. "Control" and "holdout" continues to show the unchanged bit.
```

Comment:
```suggestion
With the experiment and flag setup, you can implement it in your app. Add the flag evaluation logic so that only the "test" variant shows the changed component or code. "Control" and "holdout" continues to show the unchanged bit.
```